ui/src/webview_config.rs

```diff
@@ -4,10 +4,13 @@ use dioxus_desktop::Config;
 use warp::logging::tracing::log;
 
 pub(crate) fn webview_config() -> Config {
-    let window = crate::window_builder::get_window_builder(true, true);
-
-    let config = Config::new()
-        .with_window(window)
+    let window = crate::window_builder::get_window_builder(true);
+    let config = if cfg!(not(target_os = "macos")) {
+        Config::new().with_window(window)
```

Comment:
Does `Config` use `self` instead of `&self`? If so, maybe we could do something like

```rust
let mut config = Config::new();
if cfg!(not(target_os = "macos")) {
   config = config.with_window(window);
}
```